packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -62,31 +66,125 @@ $tc-collapsible-panel-btn-color: #555964;
 	}
 }
 
-.group {
-	justify-content: space-between;
-
-	> * {
-		margin: 0 $tc-collapsible-panel-padding-normal;
+.tc-panel {
+	:global(.panel-heading) {
+		@include header-style;
+		padding-right: $tc-collapsible-panel-padding-larger + $tc-collapsible-panel-padding-normal;
 	}
+}
 
-	:global(.label) {
-		white-space: nowrap;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		font-size: 1em;
+.tc-selectable-panel {
```

Comment:
Please try another approach : 
1. Define the common style for all headers and set this style in the header component
2. Define 2 themes with their own style that will come after the common header style
3. The default theme will be the one we have today
4. The component can take a theme as props that will set a className to apply the wanted theme

Start from a blank scss please to do that and pick only the small pieces you need. this will make the style clearer and avoid the duplications we have now.

You need to define : 
* a common header style
* a default theme header style
* a specific header style
* a common selected style